dockstore-webservice/src/main/java/io/dockstore/webservice/jdbi/WorkflowDAO.java

```diff
@@ -236,4 +237,9 @@ public WorkflowDAO(SessionFactory factory) {
     public Workflow findByAlias(String alias) {
         return uniqueResult(namedQuery("io.dockstore.webservice.core.Workflow.getByAlias").setParameter("alias", alias));
     }
+
+    public long getWorkflowIdByWorkflowVersionId(long workflowVersionId) {
+        return ((BigInteger) namedQuery("Workflow.getWorkflowIdByWorkflowVersionId")
```

Comment:
What happens if it doesn't find the id? I suspect you will get a NPE. Your code that calls this method always assumes that it finds it as well. Suggest returning a Long instead.

Reply:
Added try catch to fix